A6lib.cpp

```diff
@@ -25,7 +25,8 @@
 //
 
 A6lib::A6lib(int transmitPin, int receivePin) {
-    A6conn = new SoftwareSerial(receivePin, transmitPin, false, 1024);
+    A6conn = new SoftwareSerial(receivePin, transmitPin, false);
```

Comment:
Yep, it was against the ESP8266. I really don't want to break compatibility for it, can we use a conditional define here? I know that at least PlatformIO allows conditional defines per-platform.